apps/native/app/src/screens/notifications/notifications.tsx

```diff
@@ -251,6 +233,7 @@ export const NotificationsScreen: NavigationFunctionComponent = ({
         title={intl.formatMessage({ id: 'notifications.screenTitle' })}
         onClosePress={() => Navigation.dismissModal(componentId)}
         style={{ marginHorizontal: 16 }}
+        showLoading={loading}
```

Comment:
There's a pattern to do something like this:

```
showLoading={res.loading && !!res.data}
```

I.e. if there's no data, we have skeleton loading UI below so we don't show the spinner in that case. But we do show it when we have are showing data but are loading/refreshing new data.

Reply:
great point! Thanks, will fix 👏